spaces/S000030/README.md

```diff
@@ -3,7 +3,11 @@ uid: S000030
 name: Hilbert space
 aliases:
   - $\ell^2$
+  - $\mathbb R^\omega$
   - Fréchet space
```

Comment:
```suggestion
name: Hilbert space $\ell^2$
aliases:
  - Fréchet space $\mathbb R^\omega$
```

Reply:
Wasn't that the same as my suggestion?

Reply:
Yes, that was my intent.

Reply:
Right.  I just think it may be more instructive, especially for a newcomer, to type things in instead of clicking buttons.
(And also many little suggestions usually result in a proliferation of little commits.  Does not seem better.)

Reply:
Your sentiment doesn't track with my experience working with mathematicians on other GitHub projects, but it's out of scope to discuss it here.

Reply:
Yeah, maybe you are right.  We should have a casual discussion at the next zoom meerting.